Src/Plugins/Security/SEC1003.ReviewPotentiallySensitiveFiles.json

```diff
@@ -4,16 +4,111 @@
+          "FileNameAllowRegex": "(?i)\\.asc$",
           "ContentsRegex": ".",
           "MessageArguments": { "fileKind": "an ASCII armored file" }
+        },
+        {
+          "SubId": "MicrosoftBitLockerRecoveryKeyFile",
+          "FileNameAllowRegex": "(?i)\\.bek$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Microsoft BitLocker recovery key file" }
+        },
+        {
+          "SubId": "WindowsBitlockerDataFile",
+          "FileNameAllowRegex": "(?i)\\.fve$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Windows BitLocker full volume encrypted data file" }
+        },
+        {
+          "SubId": "JavaKeystoreFile",
+          "FileNameAllowRegex": "(?i)\\.jks$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Java keystore file" }
+        },
+        {
+          "SubId": "KDEWalletManagerDatabaseFile",
+          "FileNameAllowRegex": "(?i)\\.kwallet$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a KDE Wallet Manager database file" }
+        },
+        {
+          "SubId": "MicrosoftSQLDatabaseFile",
+          "FileNameAllowRegex": "(?i)\\.mdf$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Microsoft SQL database file" }
+        },
+        {
+          "SubId": "OpenVPNClientConfigurationFile",
+          "FileNameAllowRegex": "(?i)\\.ovpn$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "an OpenVPN client configuration file" }
+        },
+        {
+          "SubId": "P12CryptographicKeyFile",
+          "FileNameAllowRegex": "(?i)\\.p12$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic key bundle" }
+        },
+        {
+          "SubId": "PEMCryptographicPrivateKeyFile",
+          "FileNameAllowRegex": "(?i)\\.pem$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic private key" }
+        },
+        {
+          "SubId": "PFXCryptographicKeyFile",
+          "FileNameAllowRegex": "(?i)\\.pfx$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic key bundle" }
+        },
+        {
+          "SubId": "PKCS12CryptographicKeyFile",
+          "FileNameAllowRegex": "(?i)\\.pkcs12$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic key bundle" }
+        },
+        {
+          "SubId": "PuTTygenPrivateKeyFile",
+          "FileNameAllowRegex": "(?i)\\.ppk$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a PuTTYgen private key" }
+        },
+        {
+          "SubId": "PasswordSafeDatabaseFile",
+          "FileNameAllowRegex": "(?i)\\.psafe3$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a password Safe database file" }
```

Comment:
>password [](start = 47, length = 8)

I'd say something like so here. Note that I'm encapsulating the company name in apostrophes to make it clear this is the company name:

a 'Password Safe' database file